src/components/Requirements/CompletedSubReqCourse.vue

```diff
@@ -83,10 +86,10 @@ export default defineComponent({
   },
   methods: {
     onReset(): void {
```

Comment:
Nit: Might be worth renaming to onDelete() to be consistent with the other name changes from Reset to Delete